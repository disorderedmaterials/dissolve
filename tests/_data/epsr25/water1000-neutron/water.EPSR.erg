 -0.325970E+02  0.856252E+01  0.599746E-03  0.000000E+00  0.100000E+00  0.409274E-03  0.504510E-03  0.000000E+00  0.000000E+00  0.000000E+00
