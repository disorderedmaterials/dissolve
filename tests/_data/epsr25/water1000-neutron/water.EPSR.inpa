           1           0           5   0.00000000       0.00000000       0.00000000       0.00000000    
           1
         120   2.00000009E-03   1.99999996E-02
   0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000    
           2
         120   2.00000009E-03   1.99999996E-02
   0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000    
           3
         120   2.00000009E-03   1.99999996E-02
   0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000    
  0.899999976       1.00000000       1.00000000       1.00000000               1
           1           1           1
  0.21390652396225798        1.5662556363791102        2.4832020235900010       0.84048003680852168        2.0346218083053747E-002 -0.16334467724407073     
           1           1           2
   1.8595415494949947       -3.3268141051754108       0.92349521873611617        2.0346216644049734E-002  0.32432951648574865      -0.16990775784982912     
           1           2           2
   2.5771868575678050        2.5410881816480679      -0.75269088113469640      -0.16334468174149350      -0.16990776940071325       0.33956587894582813     
