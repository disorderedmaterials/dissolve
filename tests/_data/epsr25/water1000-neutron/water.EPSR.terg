 -0.217372E+02  0.000000E+00 -0.217372E+02
 OW  OW 
  0.796935E+03  0.000000E+00  0.796935E+03
 OW  HW 
 -0.163881E+04  0.000000E+00 -0.163881E+04
 HW  HW 
  0.820141E+03  0.000000E+00  0.820141E+03
