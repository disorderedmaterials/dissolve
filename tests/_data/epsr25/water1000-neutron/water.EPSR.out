
  EPSR version 25: 2016-02-24      

  Trial no.     1      Iteration no.     1      No. of configurations in sum     1
  Chi-square  0.599746E-03 R-space coeff.:  0.409274E-03 Abs. empirical energy:   0.000000E+00
  Quality factor  0.504510E-03 Control level, baseline and threshold:  0.000000E+00  0.000000E+00  0.500000E+00
  Pair energies: repulsive -0.325972E+02 harmonic  0.324465E+01 per molecule
  Tethered energy:  0.000000E+00 per molecule
  Three-body energy:  0.000000E+00 per molecule
  Three-body pressure:  0.000000E+00 kbar
  No. of .erg lines:       0 Average gradient of quality factor:  0.000000E+00
  Sizefactor    1.00000 Number density  0.10000E+00
  Net charge =    0.00000
  R-factor for data file  1:  0.8730E-03
  R-factor for data file  2:  0.1820E-03
  R-factor for data file  3:  0.7442E-03
  Noise in average g(r)  0.512780E+02
  Intermolecular energy and U_pol -0.325970E+02  0.000000E+00 kJ/mole
  Intermolecular pressure  0.856252E+01 kbar
  Average dipole moment and std.  0.230039E+01  0.156319E+00 D
  Av. intermolecular energy  0.000000E+00+- 0.000000E+00 kJ/mole
  Av. intermolecular pressure  0.000000E+00+- 0.000000E+00 kbar
  Number of rejects in MOLFIT:-
    1000       0       0       0
     736       0       0       0
